rg_instructor_analytics/templates/rg_instructor_analytics/cohort.html

```diff
@@ -0,0 +1,45 @@
+<%page args="section_data" expression_filter="h"/>
+<%
+    import uuid
+
+    from django.utils.translation import ugettext as _
+    from openedx.core.djangolib.markup import HTML
+    from mock import patch
+    from xmodule.html_module import HtmlDescriptor
+    from xblock.field_data import DictFieldData
+    from xblock.fields import ScopeIds
+    from openedx.core.lib.url_utils import quote_slashes
```

Comment:
ditto, and could you please explain why do we need all these libs.

Reply:
I delete this import because they are unused.